### PR TITLE
generated devise gem

### PR DESCRIPTION
**Pull request by EliezerAmbion**
## The Story
- Generated the devise gem
- Installed the bootstrap css framework
- Added a navbar
- Added custom field in the user table (username)

## Screenshots
nav

## References
https://www.youtube.com/watch?v=9K5YvsrKBRk

### CODE REVIEW

**Comment by jarencudilla** [comment]
Nice one!

**Review by jarencudilla — APPROVED** [review]
Thank you for setting bootstrap and devise

**Review by DanielCadiz — APPROVED** [review]

**Review by AdrianIlaga — APPROVED** [review]

File: app/views/layouts/shared/_flash.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="alert alert-info mt-4" role="alert">
+    <%= flash[:notice] %>
+  </div>
+<% end %>
+
+<% if flash[:alert] %>
+  <div class="alert alert-warning mt-4" role="alert">
+    <%= flash[:alert] %>
+  </div>
+<% end %>
```
Comment (analyn-cajocson) [inline app/views/layouts/shared/_flash.html.erb]: add new line

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+        <li class="nav-item">
+          <%= link_to 'Sign Up', new_user_registration_path, class:"nav-link" %>
+
+        </li>
+
+      </ul>
+    </div>
+  </div>
+</nav>
```
Comment (analyn-cajocson) [inline app/views/shared/_navbar.html.erb]: add new line

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**